raft/src/test/java/org/apache/kafka/raft/KafkaRaftClientTest.java

```diff
@@ -1436,6 +1438,28 @@ public void testInvalidFetchRequest() throws Exception {
         context.assertSentFetchPartitionResponse(Errors.INVALID_REQUEST, epoch, OptionalInt.of(localId));
     }
 
+    @Test
+    public void testFetchRequestVersionHandling() throws Exception {
+        int localId = 0;
+        int otherNodeId = 1;
+        int epoch = 5;
+        Set<Integer> voters = Utils.mkSet(localId, otherNodeId);
+
+        RaftClientTestContext context = RaftClientTestContext.initializeAsLeader(localId, voters, epoch);
+
+        // Latest fetch request.
```

Comment:
nit: `Latest version of the fetch request`?

Reply:
Refactored.